install_idm_server.yml

```diff
@@ -0,0 +1,146 @@
+- hosts: localhost
+  vars:
+    chassis_password: "{{ instackenv_content.nodes[0].pm_password }}"
+
+  tasks:
+    - name: add IdM host to inventory file
+      add_host:
+        name: "{{ idm_host_fqdn }}"
+        groups: "idm"
+        ansible_host: "{{ idm_host_fqdn }}"
+        ansible_user: "root"
+        ansible_ssh_private_key_file: "{{ ansible_ssh_key }}"
+        ansible_python_interpreter: "{{ python_interpreter }}"
+
+    - name: Install RHEL 8.2 on the IdM host
```

Comment:
any specific reason to use RHEL 8.2?

Reply:
RHEL > 8.3 is required to enable FIPS(Federal Information Processing Standard). Since the packages come from the OSP 16.2 rhos-release puddle, RHEL 8.4 has to be installed on the IdM host. So we install RHEL 8.2 first and upgrade to RHEL 8.4.